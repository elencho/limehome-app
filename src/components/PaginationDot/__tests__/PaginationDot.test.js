import "react-native";
import React from "react";
import renderer from "react-test-renderer";

import PaginationDot from '../PaginationDot'

it("renders correctly", () => {
  const tree = renderer.create(<PaginationDot />).toJSON();
  expect(tree).toMatchSnapshot();
});
