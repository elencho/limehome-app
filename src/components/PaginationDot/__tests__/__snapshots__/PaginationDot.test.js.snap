// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F7F1E9",
      "borderRadius": 100,
      "height": 10,
      "justifyContent": "center",
      "marginHorizontal": 10,
      "width": 10,
    }
  }
>
  <View />
</View>
`;
